### DISCUSSION TITLE
Scroll down button/icon in Player menu

### DISCUSSION THREAD

**Discussion by winklesi** [SRGSSR/pillarbox-documentation#58] [Ideas]
On PLAY web we disuss how to improve the UX on the video detailpage. An idea is to open the video in the browser fullscreen. 
To show the information of the detailpage we discussed a scroll down icon like youtube. To do this a possible solution could beto have an icon to click on in the player bar. thx for your feedback

![IMG_2093](https://github.com/SRGSSR/pillarbox-documentation/assets/46777146/c891619a-71fe-4a0c-b05f-661fbc5547fa)
![IMG_2094](https://github.com/SRGSSR/pillarbox-documentation/assets/46777146/aac4f1c7-9901-41d2-9f45-80d72810f29f)

**Comment by amtins** [comment]
Hello @winklesi , that's an excellent idea. It's possible to do this without necessarily adding a new button. All you'd have to do is overwrite the click function of the full-screen component to redefine its behavior.

The code could look something like this:

```javascript
const player = SRGLetterbox.getInstances()[0].player;

player.controlBar.fullscreenToggle.handleClick = () => {
  if (!document.fullscreenElement) {
    document.body.requestFullscreen().then(() => {
      setTimeout(() => {
        player.addClass('vjs-fullscreen');
      }, 50)

      player.el().style = `
        position: fixed;
        top: 0;
        right: 0;
        bottom: 0;
        left: 0;
      `;
    });

    return;
  }
  
  player.el().style = '';
  player.removeClass('vjs-fullscreen');
  document
    .exitFullscreen();
}
```

Note that this example focuses on the player only, and that the various animations and interactions need to be added.

**Reply by winklesi** [reply]
thank you very much!
